### ISSUE TITLE
go.cd DNS is dead, whois problems?

### ISSUE THREAD

**Issue by tedder** [gocd/gocd#3051]
##### Issue Type
- Bug Report

##### Summary

```
$ whois go.cd                                                                      
Domain Name: go.cd
Sponsoring Registrar: CONIC
>>> Last update of WHOIS database: 2016-12-28T14:13:35.555Z <<<

host go.cd
Host go.cd not found: 3(NXDOMAIN)

$ nslookup go.cd 8.8.8.8
Server:         8.8.8.8
Address:        8.8.8.8#53

** server can't find go.cd: NXDOMAIN
```

**Comment by nerro** [comment]
it's since 3 days. Information from http://whois.domaintools.com/go.cd

```
Whois Record for Go.cd
How does this work?

 Whois & Quick Stats
IP Address	54.192.119.15 - 63 other sites hosted on this server	  
IP Location	United States - California - San Francisco - Amazon.com Inc.
ASN	United States AS16509 AMAZON-02 - Amazon.com, Inc., US (registered May 04, 2000)
Whois History	36 records have been archived since 2010-07-25	  
Whois Server	whois.nic.cd
 Website
Website Title	None given.	  
Whois Record ( last updated on 2016-12-28 )

 
Warning—Incomplete Record
We believe this domain exists, but we are getting incomplete data from the Whois server.

Domain Name: go.cd
Sponsoring Registrar: CONIC
```

**Comment by arvindsv** [comment]
Yes. Take a look at https://groups.google.com/d/topic/go-cd/ssWT_krsLdM/discussion

Will probably bring up a replacement domain soon (temporarily, at least).

**Comment by arvindsv** [comment]
I'm closing this issue.

**Comment by maxlinc** [comment]
I think you should leave this open. You might have a plan of action, but this is definitely unresolved & so it's good for people coming here for information to see that it's not resolved but that you are working on it.

I think the ticket should be open until:
- https://go.cd is working again
OR
- https://github.com/gocd/gocd has had links updated to use an alternate domain (both the link on top of the page & in the README.md)

**Comment by tedder** [comment]
The reason it's a big issue (for me) is that my apt repository updates are failing now. Nontrivial.

**Comment by arvindsv** [comment]
There's an update posted [here](https://groups.google.com/d/msg/go-cd/ssWT_krsLdM/xobPCVUoEwAJ).

tl;dr Replacing go.cd with **gocd.io** everywhere should work. There are broken links which will be fixed soon.

**Comment by ketan** [comment]
go.cd is back online with redirects in place and most links should continue to work. I'm going to close this ticket out.

**Comment by dimitrovs** [comment]
@arvindsv how long did it take for Conic to bring back the domain and did they provide any explanation as to what happened? I just registered a .cd domain for a project and as it still showing NXDOMAIN despite being "active" in their control panel I started looking for Conic reviews online. That's how I stumbled upon your issue. I suppose the problem was it's a 2-letter domain? Their official policy is apparently 4+ letters, but they don't enforce it at registration, I only found out about it after registering my 3-letter .cd .

**Comment by arvindsv** [comment]
@dimitrovs It took more than two weeks (finally got back on Jan 9th, after going down on Dec 23rd). There was no real explanation for the lack of communication. They said that we had to pay some more money. They hadn't sent us the invoice.

There are some details [here](https://www.gocd.org/2017/01/05/why-we-moved-from-go-cd-to-gocd-io.html).

**Comment by arvindsv** [comment]
The fact that conic.cd and nic.cd themselves have been squatted on gives me no more confidence about their abilities. I'm happy we've moved over to gocd.org.

**Comment by dimitrovs** [comment]
Thanks. I was able to find their premium pricing page now for <4 character domains. I expect they will ask me for more money too.

**Comment by ketan** [comment]
IIRC, the issue was that CONIC had raised an additional invoice on their "management portal".

They sent no email notification to the DNS or administrative contacts for anyone to act on payment of the invoice. It was unfortunate that the incident happened during the christmas holidays and there was no real "support" available from CONIC. Emails and phone were not replied to.

The additional pricing ~~extortion~~ was for a short domain name as you mentioned.

**Comment by ketan** [comment]
FYI: We'd had that domain for quite some time, before they launched differential pricing for short domain names.

**Comment by dimitrovs** [comment]
Thanks @ketan . I wish I had read about you troubles before paying them the initial registration fee...
